gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageProvider.java

```diff
@@ -57,6 +56,7 @@ synchronized Storage getStorage(
       Function<List<AccessBoundary>, String> downscopedAccessTokenFn)
       throws IOException {
     if (!canCache(storageOptions, interceptors, pCUExecutorService)) {
+      logger.atInfo().log("Ignoring storage object cache.");
```

Comment:
should we change this to warning?

Reply:
This is expected behaviour when caching is not enabled, warning is something when things are not idle.